### PR TITLE
Update Postgres Example

### PR DESCRIPTION
**Pull request by burkeholland**

### CODE REVIEW

**Comment by burkeholland** [comment]
#sign-off per @TomArcher

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/postgresql.md](https://github.com/burkeholland/azure-docs-sdk-node/blob/patch-3/docs-ref-services/postgresql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C10%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707102006128766-42%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/postgresql.md [hunk 1]
```diff
@@ -22,31 +22,34 @@ This library is a non-blocking PostgreSQL client for Node.js, supporting pure Ja
...
- 
-  console.log('number:', res.rows[0].number);
+const pg = require('pg');
+const connectionString =
+  'postgres://{username}@{database-name}:{password}@{database-name}.postgres.database.azure.com:5432/Items?ssl=true';
+
+const client = new pg.Client(connectionString);
+client.connect();
+
+client.query('SELECT * FROM {table-name}', (err, res) => {
```
Comment (johnpapa) [inline docs-ref-services/postgresql.md]: what do you think about this?

```
const sql = 'SELECT * FROM {table-name}';
client.query(sql, (err, res) => console.log(res));
```

I think this is cleaner. Thoughts?

@burkeholland @TomArcher

Comment (burkeholland) [inline docs-ref-services/postgresql.md]: More lines, but more clear. There's a fine line between too brief and too verbose. I think in this case it works fine. I'm good with it.

Comment (johnpapa) [inline docs-ref-services/postgresql.md]: Ok. LGTM

Comment (TomArcher) [inline docs-ref-services/postgresql.md]: +1 as it makes it super easy for the customer to quickly see where to make their change.

Comment (johnpapa) [inline docs-ref-services/postgresql.md]: Burke, you got this?

Comment (johnpapa) [inline docs-ref-services/postgresql.md]: Burke, you got this?

Comment (burkeholland) [inline docs-ref-services/postgresql.md]: I've got it

**Pull request merged**